metaflow/plugins/kfp/kfp.py

```diff
@@ -1090,6 +1109,34 @@ def call_build_kfp_dag():
         kfp_pipeline_from_flow.__name__ = self.name
         return kfp_pipeline_from_flow, pipeline_conf
 
+    @staticmethod
+    def create_shared_volumes(
```

Comment:
Can we add a comment describing how a `shared_volume` is different from a normal `volume` and the use case (foreach/branching splits)?